assets/prototype/domain/eventEditor/data/mutations/prices/test/data.ts

```diff
@@ -0,0 +1,62 @@
+import { pickBy } from 'ramda';
+import { nodes as prices } from '../../../queries/prices/test/data';
+import { ReadQueryOptions } from '../../../queries/types';
+import { MutationInput, MutationType } from '../../../../../../application/services/apollo/mutations/types';
+import { ucFirst } from '../../../../../../application/utilities/text/changeCase';
+import { mutations } from '../../';
+
+export const mockedPrices = {
+	[MutationType.Create]: { ...prices[0], id: prices[0].id + '-alpha' }, // make sure to change the ID to make it different}
+	[MutationType.Update]: prices[0],
+	[MutationType.Delete]: { id: prices[0].id, __typename: prices[0].__typename },
+};
+
+export const getMutationMocks = (mutationInput: MutationInput, mutationType: MutationType) => {
+	return [
+		{
+			request: getMockRequest(mutationInput, mutationType),
+			result: getMockResult(mutationInput, mutationType),
+		},
+	];
+};
+
+export const getMockRequest = (mutationInput: MutationInput, mutationType: MutationType): ReadQueryOptions => {
+	const input: MutationInput = {
+		clientMutationId: `${mutationType}_PRICE`,
+		...mutationInput,
+	};
+	if (mutationType !== MutationType.Create && !input.id) {
+		input.id = mockedPrices[mutationType].id;
+	}
+
+	// Avoid ticketId being part of the request
+	delete input.ticketId;
```

Comment:
Not smth to be worried in tests, but worth mentioning the https://ramdajs.com/docs/#omit
in case we will need the same approach in non-testing environments

Reply:
Done :)